src/config/default-config.js

```diff
@@ -56,20 +56,10 @@ async function getAwsPoolId() {
   return poolId;
 }
 
-async function getAwsAccountId() {
-  const sts = new AWS.STS({
-    region: awsRegion,
-  });
-
-  const data = await sts.getCallerIdentity({}).promise();
-  return data.Account;
-}
-
-
 
 const config = {
   port: parseInt(process.env.PORT, 10) || 3000,
-  accountId: process.env.NODE_ENV === 'development' ? '000000000000' : '242905224710',
+  awsAccountId: process.env.AWS_ACCOUNT_ID || '000000000000',
```

Comment:
It seems that all the defaults are set for development. If this is so, I don't think we need code on line 111